contracts/src/v0.6/tests/ConcreteChainlink.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.0;
+
+import "../Chainlink.sol";
+import "../vendor/CBORChainlink.sol";
+import "../vendor/BufferChainlink.sol";
+
+contract ConcreteChainlink {
```

Comment:
Nit: At some point we decided to call things `*TestHelper` instead of `Concrete*` to make their intention more clear. I'd prefer we stick with that and call it `ChainlinkTestHelper` for consistency moving forward, same for ChainlinkClient.